Sources/MockoloFramework/Models/ThrowingKind.swift

```diff
@@ -0,0 +1,32 @@
+enum ThrowingKind: Equatable {
+    case none
+    case any
+    case `rethrows`
+    case typed(errorType: String)
+
+    var hasError: Bool {
+        switch self {
+        case .none:
+            return false
+        case .any:
+            return true
+        case .rethrows:
+            return true
+        case .typed(let errorType):
+            return errorType != .neverType && errorType != "Swift.\(String.neverType)"
+        }
+    }
+
+    var syntax: String? {
+        switch self {
+        case .none:
+            return nil
+        case .any:
+            return .throws
+        case .rethrows:
+            return .rethrows
+        case .typed(let errorType):
+            return "\(String.throws)(\(errorType))"
+        }
+    }
```

Comment:
In my opinion, this variable can move to Templates directory by creating `ThrowingKindTemplate.swift` because mockolo clearly separates object representation and rendering  into Models and Templates.

Reply:
That's a good insights. The modeling and code output have been described separately.
https://github.com/uber/mockolo/pull/267/commits/d958837be0888d8285587a42b78149280d2e1e1e